source/extensions/omni.isaac.lab/omni/isaac/lab/managers/reward_manager.py

```diff
@@ -213,6 +213,9 @@ def _prepare_terms(self):
                     f"Weight for the term '{term_name}' is not of type float or int."
                     f" Received: '{type(term_cfg.weight)}'."
                 )
+            # skip terms with weight 0
+            if term_cfg.weight == 0.0:
+                continue
```

Comment:
We removed this on purpose since many times we want to have a curriculum over rewards which may start at zero but go up during training. If you want to remove a reward term, please remove it from the config itself (or set it to None).